docs/GettingStarted.md

```diff
@@ -1,5 +1,9 @@
 # Datadog Ruby Trace Client
 
+<div class="alert alert-info">
+<strong>Beta</strong>: dd-trace-rb v2.0.0.beta1 is now available! This major release includes breaking changes, such as renaming the gem to <code>datadog</code> and extracting the CI visibility component to the <code>datadog-ci</code> gem. Please try it out and provide feedback.
```

Comment:
Should we perhaps link to https://github.com/DataDog/dd-trace-rb/blob/2.0/docs/UpgradeGuide2.md ?

Reply:
Good idea--added that in the latest commit. 👍